docs/advanced-topics/metrics.rst

```diff
@@ -12,38 +16,62 @@ Users are able to execute workflows on various platforms using Dockstore's Launc
 
 Platforms are able to submit metrics of workflows executed on their platform to Dockstore and we aggregate the metrics and display them in the UI to the users in the :ref:`Metrics tab <How to view metrics>`.
 
-.. _How to view metrics:
+Why Would I, a Platform Owner, Want to Submit Workflow Metrics?
+---------------------------------------------------------------
 
-How do I view workflow metrics?
--------------------------------
+As a platform owner, workflow metrics indicate to others that your platform is compatible with many workflows on Dockstore. Workflow metrics provide valuable information to users, including information about the resources and time needed to run the workflow. It helps the user determine if the workflow is high quality and likely to function correctly.
 
-To search for workflows with metrics, navigate to the `search <https://dockstore.org/search>`_ page and select a platform for the Execution Metrics and/or Validation Metrics facets.
+Platform owners are able to submit metrics for workflow executions, retrieve submitted executions, and update submitted executions. 
 
-.. figure:: /assets/images/docs/metrics/metrics-search-facets.png
-    :alt: Metrics search facets
+.. _Getting started with submitting metrics:
 
-Select a workflow and click on the Versions tab. Versions that have metrics have a check mark in the Metrics column.
+Getting Started With Submitting Metrics to Dockstore
+----------------------------------------------------
 
-.. figure:: /assets/images/docs/metrics/versions-metrics-column.png
-    :alt: Metrics column in Versions table
+The following sections go over how a platform can submit workflow metrics to Dockstore. 
 
-Select a version with metrics then click on the Metrics tab to view the metrics available.
+You will need to:
 
-.. figure:: /assets/images/docs/metrics/metrics-tab.png
-    :alt: Metrics tab
+.. contents::
+   :local:
+   :depth: 1
 
-Why would I, a platform owner, want to submit workflow metrics?
----------------------------------------------------------------
+Dockstore will occasionally aggregate the workflow metrics you submit and display them in the :ref:`Dockstore UI<How to view metrics>`.
```

Comment:
`occasionally` sounds a bit weird, while we work out out schedule, just drop `occasionally`